src/ui/toolbar-button.ts

```diff
@@ -0,0 +1,16 @@
+import joplin from 'api';
+import { ToolbarButtonLocation } from 'api/types';
+import constants from '../constants';
```

Comment:
I don't think it's a good pattern to import all the constants in all the files. If you import only what you need, it will make the code more readable and scalable. `import { PLUGIN_TOOLBAR_BUTTON_ID } from '../constants'`